### ISSUE TITLE
Devices do not communicate on LAN

### ISSUE THREAD

**Issue by kylemiller3** [TheLastRar/CLR-DEV9#10]
Ok I have a strange setup that I can configure in 2 different ways:
```
<-WLAN-> | Linux | <-eth-> | Windows |
```
```
<-WLAN-> | Linux | 
<-WLAN-> | Windows |
```
First setup internet is directly through computer 1.
Second is they both connect to the same WLAN.

Both of these setups work for the internet connection for the ps2 setup disk.

What I've tried: 
Hosting on Linux in config 1
1. Disabling firewall on Linux
2. Adding UDP 3658 port in Incoming ports config and enabling it 
3. Selecting adapter wlan0 in winsock mode and PC IP enabled
4. Enabled ethernet option
5. Running midnight club 3 on Linux host opening a lobby
6. Selecting adapter ethernet with default settings
7. Try to search for LAN game

Client log shows destination port is 3658
Client log shows this message over and over:
```
Error 113000 unexpected packet, dropping
```
Client game reports no servers

I've tried the same thing in config 2 with adapters changed. I've also tried windows hosting in bridged mode but it also did not work.

**Comment by TheLastRar** [comment]
I've managed to reproduce this locally, currently investigating the cause

**Comment by TheLastRar** [comment]
I've pushed a fix, you should be able to compile a functional version now.

Compiled builds for both Windows and Linux should be available later today.

**Comment by TheLastRar** [comment]
This should be fixed in v0.8.5, See https://forums.pcsx2.net/Thread-Experimental-Winsock-based-DEV9-plugin-Now-with-HDD-Support

Downloads for both windows and linux can be found here https://github.com/TheLastRar/CLR-DEV9/releases/tag/v0.8.5
